python/ql/src/semmle/python/dataflow/new/internal/DataFlowPublic.qll

```diff
@@ -352,7 +356,8 @@ class BarrierGuard extends GuardNode {
 }
 
 /**
- * A reference contained in an object. This is either a field or a property.
+ * IPA type for tracking data content associated with values.
```

Comment:
IPA type is an internal name. See https://help.semmle.com/QL/ql-handbook/types.html#algebraic-datatypes

```suggestion
 * Algebraic datatype for tracking data content associated with values.
```